docs-src/concepts/what-is-an-update.md

```diff
@@ -0,0 +1,41 @@
+---
+id: what-is-an-update
+title: What is an Update?
+sidebar_label: Update
+description: An Update is a request to and a response from Workflow Execution.
+tags:
+  - term
+  - updates
+  - explanation
+---
+
+An Update is a request to and a response from a Temporal Client to a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+
+- [How to develop, send, and handle Updates in Go](/go/updates)
+
+The Workflow must have a function to handle the Update.
+Unlike a [Signal](/concepts/what-is-a-signal) handler, the Update handler function can mutate the state of the Workflow while also returning a value to the caller.
+The Update handler listens for Updates by the Update's name.
+
+When there is the potential for multiple Updates to cause a duplication problem, Temporal recommends adding idempotency logic to your Update handler that checks for duplicates.
+
+An Update has four phases.
+
+1. **Admission.** The Temporal Cluster first validates Update submissions against the configured resource usage limits.
```

Comment:
Should we add a comment on when a workflow can/can't continue as new when there are inflight updates?

Reply:
@Quinn-With-Two-Ns Any info you have on that would be great to add!
@mmcshane for deets too?

Reply:
I believe if the update is `Admitted` and until it is `Accepted` the server will stop the workflow from continuing as new